requirements.txt

```diff
@@ -11,3 +11,4 @@ pyfakefs==3.6
 PyYAML==5.1
 simplejson==3.16.0
 six==1.11.0
+2to3==1.0
```

Comment:
Let's keep this list alphabetized, so `2to3` should be the first line in the file.